src/tests/Microsoft.PowerFx.Core.Tests/ThreadingTests.cs

```diff
@@ -34,8 +34,8 @@ public void CheckFxCore()
                 "DateAddFunction.SubDayStringList",
                 "PrettyPrintVisitor.BinaryPrecedence",
                 "ErrorResource.ErrorResourceTagToReswSuffix",
-                "StringResources.Strings",
-                "StringResources.ErrorResources",
+                "StringResources._resourceManager",
```

Comment:
>_resourceManager [](http://example.com/codeflow?start=33&length=16)

If you add a [ThreadSafeProtectedByLockAttribute] around these fields, they'll be considered safe and you don't have to add here.